lib/index.js

```diff
@@ -66,6 +67,12 @@ module.exports = options => {
           node.tag = node.attrs.tag
           delete node.attrs.tag
         }
+        // Check if should render as inline
+        if (node.attrs.hasOwnProperty('inline')) {
```

Comment:
`if (node.attrs.inline) {` will do 👍

Reply:
As `node.attrs.inline` is empty when just set as a flag (`<markdown inline>...</markdown>`) it would actually evaluate to false. If not checking `hasOwnProperty` one would be force to specify an explicit value for the attribute. Checking it this way allows for the generally expected behavior of HTML attributes, so that one can set it as both just a flag and explicitly if one would prefer, like so: `<markdown inline="1">...</markdown>`. This way, it technically also supports explicitly disabling it, `<markdown inline="0">...</markdown>`.

Reply:
You're right and I'm wrong. Sorry about that :D

Reply:
No worries :)